iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/procedure/Procedure.java

```diff
@@ -52,6 +52,7 @@ public abstract class Procedure<Env> implements Comparable<Procedure<Env>> {
   private static final Logger LOG = LoggerFactory.getLogger(Procedure.class);
   public static final long NO_PROC_ID = -1;
   public static final long NO_TIMEOUT = -1;
+  private boolean isDeserialized = false;
```

Comment:
add some comments for resatrt and leader trasnfer

Reply:
added